applications/chats/consumers.py

```diff
@@ -0,0 +1,62 @@
+from channels.db import database_sync_to_async
+from channels.exceptions import DenyConnection
+from channels.generic.websocket import AsyncJsonWebsocketConsumer
+
+
+@database_sync_to_async
+def get_room(user, room_id):
+    return user.room_set.filter(id=room_id).first()
+
+
+@database_sync_to_async
+def save_message(user, room, message):
+    return room.message_set.create(writer=user, content=message)
```

Comment:
If these functions are only used once and don't have a lot of logic why not doing it directly in the other methods? You can keep them separated if you want, just asking. Also maybe they could be moved inside the `ChatConsumer` class.

Reply:
If I create a new method, I would replace it by this `await database_sync_to_async(self.room.message_set.create)(writer=self.user, content=message)` and it looks a little weird IMHO. I agree that I should move inside the consumer class. wdyt?